src/components/get_started/getStartedBanner.vue

```diff
@@ -1,37 +1,54 @@
 <template>
-  <div id="center">
+  <div>
+    <div id="center">
     <h2 class="highlight">COMMITTED TO:</h2>
     <commitment />
+    </div>
   </div>
-
 </template>
-
 <script>
-import commitment from '@/components/home/commitment'
+import commitment from '@/components/get_started/commitment'
 
 export default {
-  name: 'homeContent',
+  name: 'getStartedBanner',
   components: {
     commitment
+  },
+  data () {
+    return {
+      activeNames: ['1']
+    }
+  },
+  methods: {
+    handleChange (val) {
+      console.log(val)
+    }
   }
 }
 </script>
 
 <style>
 .highlight {
-  color: #FFF;
+  color: #FFFFFF;
   font-size: 0.4em;
 }
 #center {
-  color: #fff;
+  color: #FFFFFF;
   font-family: 'StratumNo2';
   font-size: 5.5em;
-  align-content: center;
-  height: 2.5em;
+  justify-content: center;
+  height: 3em;
   width: 100%;
-  padding-top: 0.5em;
-
+  padding-top: 1em;
+  padding-left: 10px;
+  background: url('/images/solar.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
 }
+
+/*
+* Don't think I need this
+
```

Comment:
We could just delete the commented-out code. Git will keep a history of it if we want to refer to it later.